must-gather/collection-scripts/gather_namespaced_resources

```diff
@@ -49,13 +52,19 @@ commands_desc+=("logicalvolume")
 commands_desc+=("lvmvolumegroup")
 commands_desc+=("lvmvolumegroupsnodestatus")
 commands_desc+=("pods")
+commands_desc+=("volumesnapshot -A")
+commands_desc+=("volumesnapshotclass")
+commands_desc+=("volumesnapshotcontent")
 
 
 
 # collect yaml output of OC commands
 oc_yamls=()
 oc_yamls+=("csv")
 oc_yamls+=("installplan")
+oc_yamls+=("volumesnapshot -A")
+oc_yamls+=("volumesnapshotclass")
+oc_yamls+=("volumesnapshotcontent")
```

Comment:
Will this be created in the install namespace or the PVC's namespace?

Reply:
fixed